src/data/subtype.lean

```diff
@@ -73,15 +79,15 @@ lemma restrict_injective {α β} {f : α → β} (p : α → Prop) (h : injectiv
 h.comp coe_injective
 
 /-- Defining a map into a subtype, this can be seen as an "coinduction principle" of `subtype`-/
-def coind {α β} (f : α → β) {p : β → Prop} (h : ∀a, p (f a)) : α → subtype p :=
+@[simps] def coind {α β} (f : α → β) {p : β → Prop} (h : ∀a, p (f a)) : α → subtype p :=
```

Comment:
What new lemmas does this add?

Reply:
```lean
[simps] > adding projection
        > subtype.coind_coe : ∀ {α : Sort u_1} {β : Sort u_2} (f : α → β) {p : β → Prop}
(h : ∀ (a : α), p (f a)) (a : α), ↑(coind f h a) = f a
```

Reply:
Whats the trick for seeing that trace output?